docs/static/releasenotes.asciidoc

```diff
@@ -14,6 +15,34 @@ See also:
 * <<release-notes-xls>>
 endif::include-xpack[]
 
+[[logstash-6-2-0]]
+=== Logstash 6.2.0 Release Notes
+
+* Added support to protect sensitive settings and configuration in a https://www.elastic.co/guide/en/logstash/current/keystore.html[keystore]
+* https://www.elastic.co/guide/en/logstash/current/multiple-pipelines.html[Multiple pipelines] is now the default configuration (applicable for RPM/DEB/Docker installations)
+* Better defaults for higher throughput while under load. https://github.com/elastic/logstash/issues/8707[#8707] https://github.com/elastic/logstash/issues/8702[#8702]
+* Log files now have a default max size and are compressed when rolled (applicable for ZIP based installs) https://github.com/elastic/logstash/issues/7482[#7482]
+* Ability to specify --pipeline.id from the command line https://github.com/elastic/logstash/issues/8868[#8868]
```

Comment:
+* Added ability to specify --pipeline.id from the command line https://github.com/elastic/logstash/issues/8868[#8868]

Reply:
done with minor variant: added ticks around `--pipeline.id`